zha/zigbee/device.py

```diff
@@ -672,7 +672,7 @@ def extended_device_info(self) -> ExtendedDeviceInfo:
             **self.device_info.__dict__,
             active_coordinator=self.is_active_coordinator,
             entities={
-                unique_id: platform_entity.info_object
+                unique_id[1]: platform_entity.info_object
                 for unique_id, platform_entity in self.platform_entities.items()
```

Comment:
Maybe we can use `for (platform, unique_id), platform_entity in `self.platform_entities.items()` to avoid `unique_id[1]`?

Reply:
actually can't we just use platform_entity.unique_id?

Reply:
Agreed, `{entity.unique_id: entity.info_object for entity in self.platform_entities.values()}` is much better!

Reply:
```
           entities={
                platform_entity.unique_id: platform_entity.info_object
                for platform_entity in self.platform_entities.values()
            },
```